metals/src/main/scala/scala/meta/internal/metals/DefinitionProvider.scala

```diff
@@ -60,7 +60,7 @@ final class DefinitionProvider(
           warnings.noSemanticdb(path)
           DefinitionResult.empty
       }
-    if (fromSemanticdb.locations.isEmpty()) {
+    if (fromSemanticdb.locations.isEmpty) {
```

Comment:
okay I will keep things as they are in other places.

As a side note in scala world there is actually established convention:
https://docs.scala-lang.org/style/naming-conventions.html#parentheses
Less clear for using java in scala because you cannot forbid calling method with () but still good rule is 
side effects -> parenthesis
pure -> no parenthesis